packages/ChangesReloaded-Core.package/CRBrowser.class/instance/pickAllChangeListItems.st

```diff
@@ -0,0 +1,4 @@
+initializing
+pickAllChangeListItems
+
+	self changeListItems do: [:each | each isPicked: true]
```

Comment:
Come to think of it, `pick` and `unpick` methods on `CRChangeListItem` would also be nice

Reply:
I see where you're coming from. But isn't that one layer of abstraction too far? Just a thought, I like the idea

Reply:
As always, I guess the answer is "it depends". I think `pick` and `unpick` are two actions that are pretty fundamental and common on the CRChangeRecord, so I thought it would help to add that layer to make the code more readable. That's an opt-in of course, we don't necessarily need another layer